scripts/chce_jenkins.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Jenkins na mikrusowym porcie 
+# Autor: Maciej Loper
+
+status() {
+    echo "[x] $1"
+}
+
```

Comment:
```suggestion

status "instalacja wymaganych pakietow"
sudo apt install -y gnupg
echo
```
Bez tego pakietu weryfikacja GPG może być niemożliwa.